lib/cocoapods/installer/analyzer/target_inspection_result.rb

```diff
@@ -0,0 +1,41 @@
+module Pod
+  class Installer
+    class Analyzer
+      class TargetInspectionResult
+        # @return [TargetDefinition] the target definition, whose project was
+        #         inspected
+        #
+        attr_accessor :target_definition
+
+        # @return [Pathname] the path of the user project that the
+        #         #target_definition should integrate
+        #
+        attr_accessor :project_path
+
+        # @return [Array<String>] the uuid of the user'S targets
+        #
+        attr_accessor :project_target_uuids
+
+        # @return [Hash{String=>Symbol}] A hash representing the user build
+        #         configurations where each key corresponds to the name of a
+        #         configuration and its value to its type (`:debug` or
+        #         `:release`).
+        #
+        attr_accessor :build_configurations
+
+        # @return [Platform] the platform of the user targets
+        #
+        attr_accessor :platform
+
+        # @return [Array<String>] the architectures used by user's targets
+        #
+        attr_accessor :archs
+
+        # @return [Bool] whether frameworks are recommended for the integration
+        #         due to the presence of Swift source in the user's targets
+        #
+        attr_accessor :recommends_frameworks
```

Comment:
No, this is intentional. This is something, which wasn't propagated to UI yet. The analyzer can detect whether Swift source code is present in the user target and recommends to use `use_frameworks! `.